docker/Dockerfile

```diff
@@ -1,9 +1,24 @@
 #== Ubuntu xenial is 16.04, i.e. FROM ubuntu:16.04
 # Find latest images at https://hub.docker.com/r/library/ubuntu/
 # Layer size: big: ~130 MB
-FROM ubuntu:xenial-20181005
-ENV UBUNTU_FLAVOR="xenial" \
-    UBUNTU_DATE="20181005"
+
+# To overwrite the build args use:
+#  docker build ... --build-arg UBUNTU_DATE=20171006
+ARG UBUNTU_FLAVOR=xenial
+ARG UBUNTU_DATE=20181113
+
+#== Ubuntu xenial is 16.04, i.e. FROM ubuntu:16.04
+# Find latest images at https://hub.docker.com/r/library/ubuntu/
+# Layer size: ~122 MB
+FROM ubuntu:${UBUNTU_FLAVOR}-${UBUNTU_DATE}
```

Comment:
I think args in the from value creates issues for building inside openshift, but I can always modify by hand to workaround that

Reply:
No worries, I will revert and use the previous format.